src/sentry/utils/http.py

```diff
@@ -169,9 +170,15 @@ def is_valid_origin(origin, project=None, allowed=None):
         parsed_hostname = parsed.hostname
 
     if parsed.port:
-        parsed_netloc = '%s:%d' % (parsed_hostname, parsed.port)
+        domain_matches = (
+            '*', parsed_hostname,
```

Comment:
This formatting makes it look like the collection includes 3 items (not 4) at a glance.